### ISSUE TITLE
Allow adding `pipeline.flaggedChs `to `raw.info['bads']`

### ISSUE THREAD

**Issue by scott-huberty** [lina-usc/pylossless#72]
I imagine there is going to be disagreement on this so I want to get the conversation started:

If I run the code in `test/test_simulated.py` I get this:

![image](https://user-images.githubusercontent.com/52462026/225961406-2dbce898-1cf4-4b7f-8403-f715e9a49f63.png)

- In the automated pipeline we flag time periods as annotations starting with _`bad`. Annotations are saved directly in the file when exporting to `.edf`.
- We do not add flagged channels to raw.info['bads'], we only save them to `pipeline.FlaggedChs`

This is inconsistent and confusing. Right now the user will see some of the pipelines decision in their file and need to add extra boilerplate to see the flagged channels.

Also QCGUI does nothing with flaggedChs after loading, and MNEVisualizer only knows about raw.info['bads'], so the dashboard doesnt show the pipelines decisions on channels right now and if the user clicks on a channel, it is added to raw.info['bads'], not flaggedChs.

I think the closer we integrate with MNE here the better. I think the FlaggedChs.tsv file is necessary but lets not make everyone life harder with more boilerplate just to see decisions on their data that aren't even destructive, for the sake of purity.

**Comment by christian-oreilly** [comment]
The `add_bad` arguments would just trigger the execution of the `mark_bad_channels(...)` at the end of the pipeline. I'd make it either a boolean or a list of strings or a dictionary
`False`: Nothing gets marked as bad
`True`: Everythin gets marked as bad
`["ch_sd"]` : Mark as bad everything labeled with "ch_sd", regardless of whether it is for channels, epochs, of ICS
`{"epochs": ["ch_sd"]}` : Mark as bad every "epochs" labeld with "ch_sd"  (not epochs really, but time windows).
This input could be passed directly to the `kinds` argument of the `mark_bad_channels()` function.

**Comment by scott-huberty** [comment]
Yeah I think this is a nice solution. Glad we could find some common ground 😉

**Comment by Andesha** [comment]
is there any updates to this now that flagging has changed formats slightly? @scott-huberty
